client/shipping-label/state/reducer.js

```diff
@@ -422,21 +422,24 @@ reducers[ SET_PACKAGE_TYPE ] = ( state, { packageId, boxTypeId } ) => {
 	const newPackages = { ...state.form.packages.selected };
```

Comment:
I don't know if it makes sense to do in this PR or even soon in a new related PR, but `newPackages` should be an array and not an object of objects. Why? For easier lookup, and manipulation. Adding this to the list of things we should review: p2-p7bbVw-1Bj#comment-2697 

#1088